pkg/runtime/runtime.go

```diff
@@ -229,3 +216,16 @@ func TemplateSpecApply[A any](info *Info) (*A, bool) {
 	spec, ok := info.TemplateSpec.ObjApply.(*A)
 	return spec, ok
 }
+
+func (i *Info) FindContainerByPodSetContainerName(psName, containerName string) *Container {
+	for psIdx, ps := range i.TemplateSpec.PodSets {
+		if ps.Name == psName {
```

Comment:
It would be nice if we could add the clarification comment what is PodSet name here: https://github.com/kubeflow/trainer/blob/a3cced015d79d2bbaba0f7e85e4335db5b7c6fbf/pkg/runtime/runtime.go#L61

I guess, we didn't name it as JobName, to allow use this structure for other runtime that don't use JobSet and Job CRDs  right?

Reply:
> It would be nice if we could add the clarification comment what is PodSet name here:

Sure.

> I guess, we didn't name it as JobName, to allow use this structure for other runtime that don't use JobSet and Job CRDs right?

Yes, the PodSet concept is a set of PodSpec. I do not want to bring the JobSet or Job concepts to runtime.